guides/documentation.md

```diff
@@ -0,0 +1,31 @@
+# Documenting your work
+
+Good development practices include writing and maintaining documentation that covers the software being written.
+
+The Write the Docs [Documentation Guide](https://www.writethedocs.org/guide/) provides an excellent writeup on [why writing documentation is important](https://www.writethedocs.org/guide/writing/beginners-guide-to-docs/#why-write-docs), best processes for writing and maintaining docs, and tips for building a documentation culture in an organization.
+
+The WTD [Documentation Principles](https://www.writethedocs.org/guide/writing/docs-principles/) are also recommended reading and should be kept in mind when writing documentation at CFPB.
```

Comment:
What would you think about putting this before the links to their guide homepage and the why-it's-important writeup? If I want people to read one thing before tl;dr-ing out, it's the principles.

Reply:
Thanks, that's a good suggestion. Implemented in 429a507.